.chloggen/google-secret-manager-provider-alpha.yaml

```
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Other alpha component changelogs seem to use `new_component` so we should use that here too I think.
```suggestion
change_type: new_component
```

---

https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/39858/files#diff-7be4ceadde7e12a7e4b01c82747d112cc2deed10b21805a23eb3278e8add8e95 I found this PR, it used `enhancement`

---

If the changelog from the previous PR is `new_component` then we can leave this one as enhancement.

---

https://github.com/open-telemetry/opentelemetry-collector-contrib/commit/49a05ab24dcaf2aa5cd629e1cd9eeb5839169bd9#diff-824a70bad7774ba59c7f93d6564ad3810cf905250bba9c63c1f7577e9bf6dac5R2 If you are referring to this PR, then yes.